src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,29 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String WORD_STARTS_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        Path inputFilePath = Paths.get(fileName);
+        ArrayList<String> inputFile = new ArrayList<>();
+        try {
+            inputFile = (ArrayList<String>) Files.readAllLines(inputFilePath);
+        } catch (IOException e) {
+            System.out.println("File does not exist");
+        }
+        ArrayList<String> result = new ArrayList<>();
+        for (String string : inputFile) {
+            String[] words = string.split(" ");
+
+            for (String word : words) {
+                if (word.replaceAll("\\W", "").toLowerCase().startsWith(WORD_STARTS_WITH)) {
+                    result.add(word.replaceAll("\\W", "").toLowerCase());
```

Comment:
let's first modify the `word` String, and then check if it starts with `WORD_STARTS_WITH`

Reply:
done